fda-model/compute/ComputeCompleteMessage.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace compute
+{
+    public class ComputeCompleteMessage : HEC.MVVMFramework.Base.Interfaces.IMessage
+    {
+        private double _iterations;
+        private int _impactAreaID;
+        public string Message
+        {
+            get
+            {
+                return $"The compute for the impact area with ID {_impactAreaID} has finished after {_iterations} iterations";
```

Comment:
This is cool. I've not seen a string structured like that before in C#, but it makes sense.

Reply:
I think it is called string interpolation